admin/code/LeftAndMain.php

```diff
@@ -206,7 +206,7 @@ public function getCombinedClientConfig() {
 	 */
 	public function getClientConfig() {
 		return [
-			'route' => Config::inst()->get($this->class, 'url_segment')
+			'route' => $this->Link()
```

Comment:
Yes; getClientConfig() affects the code in LeftAndMain.js which was updated in the same pull request.